rollup.config.js

```diff
@@ -0,0 +1,51 @@
+import babel from 'rollup-plugin-babel';
+import { list as babelHelpersList } from 'babel-helpers';
+import uglify from 'rollup-plugin-uglify';
+import { minify } from 'uglify-es';
```

Comment:
not so sure that we should pass custom uglify to the plugin, `uglify-es` is not yet considered stable enough and `rollup-plugin-uglify` has its dependency on `uglify-js@3`